static/app/views/organizationGroupDetails/groupActivityItem.tsx

```diff
@@ -140,6 +141,35 @@ function GroupActivityItem({activity, orgSlug, projectId, author}: Props) {
               author,
             });
       case GroupActivityType.SET_RESOLVED_IN_COMMIT:
+        const deployed_releases: Array<BaseRelease> = [];
+        for (const release of activity.data.commit.releases) {
```

Comment:
we should safely access this via `activity.data.commit?.releases` since the commit property in data might not be included